Automattic/newspack-plugin
# Campaigns Wizard: "view as" UI

### All Submissions:






### Changes proposed in this Pull Request:



_Note: merge after https://github.com/Automattic/newspack-popups/pull/349 is merged_

### How to test the changes in this Pull Request:

1. Switch to branch `add/view-from-group` (https://github.com/Automattic/newspack-popups/pull/349) of `newspack-popups`
2. Create at least two groups, at least two segments, and some related campaigns*
2. Visit the Campaigns Wizard, observe a new tab: "Preview", open it
3. Try out the previews in different configurations of segments and groups
4. Observe that the preview configuration is persisted after a page reload

\* My setup was: "subscribed", "unsubscribed" segments; "newsletter", "election" campaign groups; two "Newsletter" campaigns – a newsletter form segmented to unsubscribed and a "thank you" message for subscribers, an "Election" campaign

### Other information:
